scripts/release.py

```diff
@@ -359,6 +361,10 @@ def prs_by_email(start_ref, end_ref):
     The dictionary is alphabetically ordered by email address
     The pull request list is ordered by merge date
     """
+    # `emails` maps from other_emails to primary email, based on people.yaml.
+    people = yaml.safe_load(requests.get(PEOPLE_YAML).text)
+    emails = { em: person['email'] for person in people.itervalues() for em in person.get('other_emails', []) }
```

Comment:
@adampalay That would be a good addition.  The script is already getting all the user details from GitHub, because it gets the details about each pull request. But that was too big a change for me to make now.  This is a low-tech solution that will improve things.